hls4ml/backends/vivado/vivado_backend.py

```diff
@@ -23,12 +23,39 @@ def __init__(self):
         self._register_flows()
 
     def _register_layer_attributes(self):
-        extended_attrs = {
-            SimpleRNN: [Attribute('recurrent_reuse_factor', default=1), Attribute('static', value_type=bool, default=True)],
-            LSTM: [Attribute('recurrent_reuse_factor', default=1), Attribute('static', value_type=bool, default=True)],
-            GRU: [Attribute('recurrent_reuse_factor', default=1), Attribute('static', value_type=bool, default=True)],
-        }
-        self.attribute_map.update(extended_attrs)
+        # Add RNN-specific attributes, recurrent_reuse_factor and static implementation
+        rnn_layers = [
+            SimpleRNN, LSTM, GRU,
+        ]
+
+        for layer in rnn_layers:
+            attrs = self.attribute_map.get(layer, [])
+            attrs.append(ConfigurableAttribute('recurrent_reuse_factor', default=1))
+            attrs.append(ConfigurableAttribute('static', value_type=bool, default=True))
+            self.attribute_map[layer] = attrs
+        
+        # Add ParallelizationFactor to Conv1D/2D
+        pf_layers = [
+            Conv1D, Conv2D,
```

Comment:
Should `PointwiseConv1D` and `PointwiseConv2D` from `hls4ml.backends.fpga.fpga_layers` be in this list (and ditto in other places)? Or how should they be handled? 

I know currently in the `main` branch they're not really doing anything, but this is a layer that I'm using in the Jet ID GNN, which in our custom branch *can* have a different behavior

Reply:
Lest there are bugs, those two layers will get a RF and PF because they inherit from Conv1D/2D. In general, the classes inherit the attributes of the superclass and only need to define new attributes.

Reply:
ok, i confirm this is the case